pruning_implementation/pruning.py

```diff
@@ -0,0 +1,69 @@
+import babeltrace
+from babeltrace import CTFWriter
```

Comment:
i think importing a module is redundant if you already imported the package.

Reply:
It is not, we need it for the CTFWriter

Reply:
OK, I thought you would get CTFWriter when you did `import babeltrace` already.

Reply:
I thought so too, but it doesn't work that way in babeltrace